src/models/attributes.ts

```diff
@@ -72,7 +72,7 @@ class AttributeObject extends Attribute {
       return val;
     }
 
-    if (this.itemClass.prototype instanceof RestfulModel) {
+    if (this.itemClass.constructor.name == 'RestfulModel') {
```

Comment:
I'm not a big fan of this, but I see why you had to do it

Reply:
@ozsivanov I'm not a big fan of it either, I'm actually working on an alternative since I assigned it to you, I'll message you when I have it!

Reply:
@ozsivanov It's not _that_ much better but I think it's more accurate and it prevents scenarios like `Label` where its superclass is not `RestfulModel`, but `Folder` which extends `RestfulModel` (two classes removed).